src/iotjs_binding.c

```diff
@@ -167,7 +167,7 @@ void iotjs_jval_destroy(iotjs_jval_t* jval) {
 
 
 static void iotjs_jval_destroy_norelease(iotjs_jval_t* jval) {
-  IOTJS_VALIDATED_STRUCT_DESTRUCTOR(iotjs_jval_t, jval);
+  IOTJS_VALIDATED_STRUCT_DESTRUCTOR_VALIDATE(iotjs_jval_t, jval);
```

Comment:
Let's go with ``IOTJS_VALIDATABLE_STRUCT_DESTRUCTOR_VALIDATE``.
If we find better idea, then rename it in another PR.